GameData/RealismOverhaul/Engine_Configs/XLR11_Config.cfg

```diff
@@ -3,31 +3,47 @@
 //
 //	Manufacturer: Reaction Motors
 //
-//	=================================================================================
-//	XLR11-RM-3
+//  =================================================================================
+//	XLR11-RM-3 // [A] Assuming this version is the RMI 6000-C4 based off tertiary sources, not listed
 //	X-1
 //
-//	Dry Mass: 95 Kg
-//	Thrust (SL): ??? kN
-//	Thrust (Vac): 28.80 kN
-//	ISP: 190 SL / 210 Vac
-//	Burn Time: 150
+//	Dry Mass: 150 Kg
+//	Thrust (SL): 24.6 kN
+//	Thrust (Vac): 26.68 kN
+//	ISP: 209 SL / 226.6 Vac
```

Comment:
Wait, so the pressure fed version has a _higher_ specific impulse than the pump-fed versions? That seems very strange.

Reply:
Frankly, I don't know either. Sutton does not give stats for turbopump fed versions of the XLR11 (XLR35 not included) despite even including a diagram of the pump used, so I simply left them unchanged.

Reply:
Well, regardless they shouldn't be _lower_ than the pressure-fed version. Probably worth bumping the -5 up a bit and the -13 up a bit more from the increased flow and chamber pressure. Something like 228 for the -5 and 229, 230 for the -13-8K and -10K respectively?